### PR TITLE
[EOS-2395] New label in marathon-lb to specify certs location

### PR DESCRIPTION
**Pull request by alvarofernandez2**

### CODE REVIEW

File: marathon_lb.py [hunk 1]
```diff
@@ -1307,7 +1307,7 @@ def get_health_check(app, portIndex):
 healthCheckResultCache = LRUCache()
 
 
-def get_apps(marathon, apps=[]):
+def get_apps(marathon, apps=[], groups=False):
```
Comment (unai-ttxu) [inline marathon_lb.py]: I think its better to define function as `get_apps(marathon, apps=[], groups=None):`

File: marathon_lb.py [hunk 2]
```diff
@@ -1574,12 +1634,12 @@ def regenerate_config(marathon, config_file, groups, bind_http_https,
                       ssl_certs, templater, haproxy_map):
     domain_map_array = []
     app_map_array = []
-    apps = get_apps(marathon)
+    apps, apps_id_label_list = get_apps(marathon, [], groups)
```
Comment (unai-ttxu) [inline marathon_lb.py]: If function `get_apps` is defined as `get_apps(marathon, apps=[], groups=None):`, I think we can refer to this function as follows:

`apps, apps_id_label_list = get_apps(marathon, groups=groups)`

**Comment by unai-ttxu** [comment]
LGTM

File: marathon_lb.py [hunk 3]
```diff
@@ -1522,7 +1522,39 @@ def get_apps(marathon, apps=[]):
...
+        for marathon_app in marathon_apps:
+            
+            if str(marathon_app.appId) == os.environ.get("MARATHON_APP_ID"):
+                continue
+
+            for service in list(marathon_app.services.values()):
+                if service.haproxy_groups:
+                    if not has_group(groups, service.haproxy_groups):
+                        continue
+                else:
```
Comment (marquetemb) [inline marathon_lb.py]: this should be an elif

**Pull request merged**